src/internal/m365/collection/drive/collections.go

```diff
@@ -147,7 +147,11 @@ func deserializeMetadata(
 				// these cases. We can make the logic for deciding when to continue vs.
 				// when to fail less strict in the future if needed.
 				if err != nil {
```

Comment:
this is the else part of an if/else, without the else.  Since you're touch this code, would you mind updating the code here and above?  Either to remove the `if err == nil` and kkeep this conditional, or to keep the continue-on-nil and remove this conditional instead.  We don't need both.